### PR TITLE
[PHEE-380] Added ngrok to expose the wiremock server

### PR DESCRIPTION
**Pull request by truthfool**
## Description

- Added ngrok to expose the wiremock server
- Script to export the URL as env variable

https://fynarfin.atlassian.net/browse/PHEE-380?atlOrigin=eyJpIjoiNzdmYjRjMjc0NjYyNDBkNTgwZDk1NWQyNjBiZjFjOTciLCJwIjoiamlyYS1zbGFjay1pbnQifQ

 _(Ignore if these details are present on the associated JIRA ticket)_

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Design related bullet points or design document link related to this PR added in the description above.

- [ ] Updated corresponding Postman Collection or Api documentation for the changes in this PR.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [ ] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

### CODE REVIEW

**Comment by truthfool** [comment]
@fynmanoj

File: .circleci/config.yml [hunk 1]
```diff
@@ -61,10 +61,17 @@ jobs:
...
+      - run:
+          name: Start ngrok tunnel
+          command: |
+            ngrok http 53013 > /dev/null & sleep 5
+            CALLBACK_URL=$(curl --silent --max-time 10 --connect-timeout 5 \
+                            --show-error http://127.0.0.1:4040/api/tunnels | \
+                            sed -nE 's/.*public_url":"https:..([^"]*).*/\1/p')
+            export CALLBACK_URL="https://$CALLBACK_URL"
+            echo CALLBACK_URL
+      - run: ./gradlew test -Dcucumber.filter.tags="@gov"
```
Comment (fynmanoj) [inline .circleci/config.yml:74]: add this command as part of above command.

**Review by fynmanoj — CHANGES_REQUESTED** [review]

**Review by fynmanoj — APPROVED** [review]

File: .circleci/ngrok-run.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```
[the remark this answers is not in the document]
Comment (truthfool) [reply .circleci/ngrok-run.sh]: done

File: .circleci/config.yml (hunk 1 above)
Comment (truthfool) [reply .circleci/config.yml]: done

File: .circleci/config.yml [hunk 3]
```diff
@@ -61,10 +61,20 @@ jobs:
...
+      - run: curl -s https://ngrok-agent.s3.amazonaws.com/ngrok.asc | sudo tee /etc/apt/trusted.gpg.d/ngrok.asc >/dev/null && echo "deb https://ngrok-agent.s3.amazonaws.com buster main" | sudo tee /etc/apt/sources.list.d/ngrok.list && sudo apt update && sudo apt install ngrok
+      - run:
+          name: Start ngrok tunnel
+          command: |
+            ngrok http 53013 > /dev/null & sleep 5
+            NGROK_URL=$(curl --silent --max-time 10 --connect-timeout 5 \
+                            --show-error http://127.0.0.1:4040/api/tunnels | \
+                            sed -nE 's/.*public_url":"https:..([^"]*).*/\1/p')
+            export NGROK_PUBLIC_URL="https://$NGROK_URL"
+            echo NGROK_PUBLIC_URL
```
[the remark this answers is not in the document]
Comment (truthfool) [reply .circleci/config.yml]: done

**Pull request merged**